index.php

```diff
@@ -11,6 +11,20 @@
 	</script>
 	<meta name="google" value="notranslate" />
 	<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
+	<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
```

Comment:
I'd probably move this to main.js.  The username to monitor should also be defined in config.js, as should the limit.